langchain4j-scoring/src/main/test/java/dev/langchain4j/model/scoring/onnx/OnnxScoringModelIT.java

```diff
@@ -0,0 +1,69 @@
+package dev.langchain4j.model.scoring.onnx;
+
+import ai.onnxruntime.OrtSession;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.output.Response;
+import dev.langchain4j.model.scoring.ScoringModel;
+import dev.langchain4j.model.scoring.onnx.OnnxScoringModel;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.data.Percentage.withPercentage;
+
+class OnnxScoringModelIT {
+
+    @Test
+    void should_score_single_text() {
+
+//        String pathToModel = "D:\\github\\langchat\\model\\reranker\\bge-reranker-large\\onnx\\model.onnx";
+//        String pathToTokenizer = "D:\\github\\langchat\\model\\reranker\\bge-reranker-large\\tokenizer.json";
+//        ScoringModel model = new OnnxScoringModel(pathToModel, new OrtSession.SessionOptions(), pathToTokenizer, 510,  true);
+
+        String pathToModel = "D:\\github\\langchat\\model\\reranker\\bge-reranker-v2-m3\\onnx\\model.onnx";
+        String pathToTokenizer = "D:\\github\\langchat\\model\\reranker\\bge-reranker-v2-m3\\tokenizer.json";
+        ScoringModel model = new OnnxScoringModel(pathToModel, new OrtSession.SessionOptions(), pathToTokenizer, 8000,  true);
+
+        String text = "The giant panda (Ailuropoda melanoleuca), sometimes called a panda bear or simply panda, is a bear species endemic to China.";
+        String query = "What is panda?";
+
+        // when
+        Response<Double> response = model.score(text, query);
+
+        // then
+        assertThat(response.content()).isCloseTo(0.99, withPercentage(1));
```

Comment:
please set `normalize=false` and add assertions with expected scores from [HF](https://huggingface.co/BAAI/bge-reranker-v2-m3)
```
scores = reranker.compute_score([['what is panda?', 'hi'], ['what is panda?', 'The giant panda (Ailuropoda melanoleuca), sometimes called a panda bear or simply panda, is a bear species endemic to China.']])
print(scores) # [-8.1875, 5.26171875]
```

Reply:
I will add this